screen/loans/src/main/java/com/ivy/loans/loan/LoansScreen.kt

```diff
@@ -78,7 +77,7 @@ fun BoxWithConstraintsScope.LoansScreen(screen: LoansScreen) {
 @Composable
 private fun BoxWithConstraintsScope.UI(
     state: LoanScreenState,
-    onEventHandler: (LoanScreenEvent) -> Unit = {},
+    onEventHandler: (LoanScreenEvent) -> Unit = {}
```

Comment:
Add screenshot tests for all cases of this UI (regular, tabular, empty state)

Reply:
Done